README.md

```diff
@@ -1,29 +1,68 @@
-# Gnareact
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-The Gnar's React/Redux app starter
+## Available Scripts
 
-## Setup
+In the project directory, you can run:
 
-Initial setup was completed with slight modifications following Andrew Farmer's
-[build your own starter](http://andrewhfarmer.com/build-your-own-starter/#0-intro)
+### `npm start`
```

Comment:
From the `yarn.lock` it looks like this project uses `yarn` instead of `npm`.

Reply:
This is the default README from create-react-app. It used to use yarn instead of npm. Looks like I need to remove the `yarn.lock` file

Reply:
NPM4LIFE